frontend/kick-covid-19/src/pages/Blog.js

```diff
@@ -21,11 +21,14 @@ class Blog extends React.Component {
   }
 
   render() {
+    console.log(
+      `[DEBUG] this.state.posts = ${JSON.stringify(this.state.posts, null, 2)}`
+    );
     return (
       <div className="row">
-        {this.state.posts.map((body, posts, id) => (
-          <Link key={posts.id} to={`/posts/${posts.id}`}>
-            <Post body={body} key={id} />
+        {this.state.posts.map(({ _id, body, answers }) => (
+          <Link key={_id} to={`/posts/${_id}`}>
+            <Post body={body} />
             <pre>{JSON.stringify(posts.body, null, 2)}</pre>
```

Comment:
you removed posts from L29 (it never was actually defined to begin with, but now the variable isn't even created) so that is why you are getting an error here

Reply:
you probably just want to delete this line and render your post component but if you want to leave this for debugging purposes maybe just do this to see the id and body text of each post
```suggestion
            <pre>{JSON.stringify({_id, body}, null, 2)}</pre>
```